docs/compilation/multi_parameters.md

```diff
@@ -1,14 +1,17 @@
 # Multi parameters
+This document explains the implications and configuration of multi parameters in **Concrete**.
 
-Integers in Concrete are encrypted and processed according to a set of cryptographic parameters. By default, multiple sets of such parameters are selected by the Concrete Optimizer. This might not be the best approach for every use case, and there is the option to use mono parameters instead.
+In **Concrete**, integers are encrypted and processed based on a set of cryptographic parameters. By default, the **Concrete** Optimizer selects multiple sets of these parameters, which may not be optimal for every use case. In such cases, you can choose to use mono parameters instead.
 
-When multi parameters are enabled, a different set of parameters are selected for each bit-width in the circuit, which results in:
+When multi parameters are enabled, the system selects a different set of parameters for each bit-width in the circuit. This approach has several implications:
```

Comment:
What is "the system"? That is not clear for me, we should say "the optimizer" here.

Reply:
Corrected, thx